opsdroid/database/mongo/__init__.py

```diff
@@ -64,10 +65,18 @@ async def put(self, key, data):
 
         """
         _LOGGER.debug("Putting %s into MongoDB.", key)
-        if "_id" in data:
-            await self.database[key].update_one({"_id": data["_id"]}, {"$set": data})
-        else:
-            await self.database[key].insert_one(data)
+        if isinstance(data, str):
+            data = {"value": data}
+        if "_id" not in data:
+            data["_id"] = key
```

Comment:
Did a little data manipulation for simplicity and consistency prior to writing to db. This will make the API as simple and correct as possible.